gabbi/case.py

```diff
@@ -20,16 +20,15 @@
 from collections import OrderedDict
 import copy
 import functools
+import http.cookies
```

Comment:
perhaps a little simpler:

```suggestion
from http import cookies
```

(would require corresponding adjustments further down, of course)